3party/protobuf/config.h

```diff
@@ -1,152 +0,0 @@
-/* config.h.  Generated from config.h.in by configure.  */
-/* config.h.in.  Generated from configure.ac by autoheader.  */
-
-/* the name of <hash_map> */
-#define HASH_MAP_CLASS unordered_map
```

Comment:
And now it's not used 😄 

There is no `configure.ac` in protobuf right now. So, I guess, it automatically configures itself during the compile-time.

Reply:
CMake generates all necessary defines from CMakeLists.txt file. It would be great to make sure that they all are properly passed to non-cmake builds (XCode).